### DISCUSSION TITLE
Help with creating a pull request validation for Doge on sol

### DISCUSSION THREAD

**Discussion by NZDAaron** [community/community#126708] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I keep getting Pull request creation failed. Validation failed: must be a collaborator , when trying to validate my token 

can someone please help :(

**Comment by chloecarlo32** [comment]
Looking for help with creating a pull request validation for Doge on Sol? Look no further! Our team at [best assignment Writing Services UK](https://ukassignmentwritershelp.co.uk/) is here to provide expert assistance. Let's ensure your project meets the highest standards. Reach out to us today for top-notch support!

**Reply by ebndev** [reply]
Hi @chloecarlo32, thanks for being a part of the GitHub Community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
